arrays/arrays.c

```diff
@@ -19,24 +19,26 @@ typedef struct Array {
 /*****
  * Allocate memory for a new array
  *****/
-Array *create_array (int capacity) {
+Array *create_array(int capacity)
+{
   // Allocate memory for the Array struct
-    Array * arr = malloc(sizeof(Array));
+  Array *arr = malloc(sizeof(Array));
+
   // Set initial values for capacity and count
-    arr->capacity = capacity;
-    arr->count = 0;
+  arr->capacity = capacity;
+  arr->count = 0;
 
   // Allocate memory for elements
-    arr->elements = calloc(capacity, sizeof(char *));
+  arr->elements = calloc(capacity, sizeof(char *));
```

Comment:
perfect :) don't tell anyone but I think you're the only one that got it right on the first try :p